test/test_functions

```diff
@@ -1983,7 +2026,7 @@ purge_disk_cache() {
 try_automount() {
```

Comment:
`try_automount()` is a misleading name here because the function either tries to *auto mount* or mounts manually. However when reading `try_automount()` I would expect it to "try to automount" or do nothing on failure. What about renaming it to: `try_automount_or_manual_mount()`?